src/App.js

```diff
@@ -10,21 +10,24 @@ import {
   Signup,
 } from "./pages";
 import { Navbar } from "./components";
+import { AuthProvider } from "./context";
 function App() {
```

Comment:
Use the arrow function here also as you have used arrow function in  your another components